docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md

```diff
@@ -0,0 +1,30 @@
+Basic Install on Windows
+========================
+
+You can locally install the SDK from the compiled Open Enclave tree by specifying
+the install-prefix to the cmake call before calling "ninja install".
```

Comment:
Formatting: Consider wrapping commands in `` so they are highlighted like code: `ninja install`